### PR TITLE
matheus/tabs support

### PR DESCRIPTION
**Pull request by zoedsoupe**
- updated screenshot
- updated readme
- added support for kitty tabs

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -2,3 +2,4 @@
 # https://help.github.com/en/articles/about-code-owners
 
 *            @gabrielcaussi
+*            @Mdsp9070
```
Comment (jpedroschmitz) [inline .github/CODEOWNERS]:
```suggestion
*            @gabrielcaussi @Mdsp9070
```

File: README.md [hunk 2]
```diff
@@ -38,6 +38,11 @@ This theme is maintained by the following person(s) and a bunch of [awesome cont
 | ---------------------------------------------------------------------------------------------------- |
 | [Gabriel Caussi](https://github.com/gabrielcaussi)                                                   |
 
+
+| [![Matheus Pessanha](https://github.com/Mdsp9070.png?size=100)](https://github.com/Mdsp9070) |
+| -------------------------------------------------------------------------------------------- |
+| [Matheus Pessanha](https://github.com/Mdsp9070)                                              |
+
```
Comment (jpedroschmitz) [inline README.md:40]: To align the table correctly, replace the content with the code below.
```
| [![Gabriel Caussi](https://github.com/gabrielcaussi.png?size=100)](https://github.com/gabrielcaussi) | [![Matheus Pessanha](https://github.com/Mdsp9070.png?size=100)](https://github.com/Mdsp9070) |
| ---------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
| [Gabriel Caussi](https://github.com/gabrielcaussi)                                                   | [Matheus Pessanha](https://github.com/Mdsp9070)                                              |
```

**Review by jpedroschmitz — CHANGES_REQUESTED** [review]
Hey, leaving my contribution here 💜 

Just some minor changes to follow the Omni template pattern =D

**Comment by zoedsoupe** [comment]
Oh, of course! I misunderstood these conventions before haha

**Review by jpedroschmitz — APPROVED** [review]
It looks great to me. 

Let's wait for @gabrielcaussi review. 💜

**Review by gabrielcaussi — APPROVED** [review]
It's perfect, thanks for the contribution!

**Pull request merged**